README.md

```diff
@@ -4,123 +4,32 @@
-
-## Development
-
-### Editing source files
-
-Most of the code is in go templates: helmfile's `*.gotmpl` and helm chart's `templates/*.yaml`. Please become familiar with it's intricacies by reading our [special section on go templating](./docs/GO_TEMPLATING.md).
-
-For editing the `values-schema.yaml` please refer to the [meta-schema documentation](./docs/meta-schema-validation.md).
-
-For working with `bats` and adding tests to `bin/tests/*` please refer to the [online bats documentation](https://bats-core.readthedocs.io/en/latest/)
-
-You can define OPA policies in `policies/*/src.rego` and unit tests in `policies/*/src_test.rego` files that are used both for statical analysis (also at build time), as well as by [gatekeeper](https://github.com/open-policy-agent/gatekeeper) (at run time) to check whether manifests are conformant. Please read [more about our setup in the docs](./docs/policies.md).
-
-### 1. Validating changes
-
-For the next steps you will need to export `ENV_DIR` to point to your values folder, and source the aliases:
-
-```bash
-# assuming you created otomi-values repo next to this:
-export ENV_DIR=$PWD/../otomi-values
-. bin/aliases
-```
-
-### Input
-
-Start by validating the configuration values against the `values-schema.yaml` with:
-
-```bash
-# all clusters
-otomi validate-values
-# For the next step you will also need to export`CLOUD` and `CLUSTER`, as it is only validating a configured target cluster:
-otomi validate-values: 1
-```
-
-Any changes made to the meta-schema will then also be automatically validated.
-
-### Output
-
-You can check whether resulting manifests are conform our specs with:
-
-```bash
-# all clusters
-otomi validate-templates
-# For the next step you will also need to export`CLOUD` and `CLUSTER`, as it is only validating a configured target cluster:
-export CLOUD=google CLUSTER=demo
-otomi validate-templates 1
-```
-
-This will check whether any CRs are matching their CRDs, but also check for k8s manifest best practices using [kubeval](https://www.kubeval.com).
-
-And to run the policy checks run the following:
-
-```bash
-# all clusters
-otomi check-policies
-# For the next step you will also need to export`CLOUD` and `CLUSTER`, as it is only validating a configured target cluster:
-otomi check-policies 1
 ```
-
-### 2. Diffing changes
-
-To test changes in code against running clusters you will need to export at least `ENV_DIR`, `CLOUD` and `CLUSTER` and source the aliases:
-
-After changing code you can do a diff to see everything still works and what has changed in the output manifests:
-
-```bash
-otomi diff
-# or target one release:
-otomi diff -l name=prometheus-operator
+helm repo add otomi https://otomi.io/otomi-core
+helm repo update
+helm install otomi otomi/otomi
```

Comment:
They  can not install otomi chart without providing values. so it should be `helm install -f values.yaml otomi otomi/otomi`